src/gstools/variogram/estimator.pyx

```diff
@@ -151,6 +154,8 @@ cdef _estimator_func choose_estimator_func(str estimator_type):
     cdef _estimator_func estimator_func
     if estimator_type == 'm':
         estimator_func = estimator_matheron
+    elif estimator_type == 'f':
```

Comment:
Maybe the char 'a' for absolute might be more intuitive or what is the reason you chose 'f'?

Reply:
It from the name `f-madogram`, since the `matheron` estimator already took, `m`.

Here is a paper using the estimator.
![paper](https://core.ac.uk/download/pdf/82449928.pdf)